variables.tf

```diff
@@ -1,88 +1,8 @@
+# IMPORTANT: Add addon specific variables here
 variable "enabled" {
   type        = bool
   default     = true
-  description = "Variable indicating whether deployment is enabled"
-}
-
-variable "cluster_identity_oidc_issuer" {
-  type        = string
-  description = "The OIDC Identity issuer for the cluster"
-}
-
-variable "cluster_identity_oidc_issuer_arn" {
-  type        = string
-  description = "The OIDC Identity issuer ARN for the cluster that can be used to associate IAM roles with a service account"
-}
-
-variable "helm_chart_name" {
-  type        = string
-  default     = "external-dns"
-  description = "Helm chart name to be installed"
-}
-
-variable "helm_chart_version" {
-  type        = string
-  default     = "6.5.6"
-  description = "Version of the Helm chart"
-}
-
-variable "helm_release_name" {
-  type        = string
-  default     = "external-dns"
-  description = "Helm release name"
-}
-variable "helm_repo_url" {
-  type        = string
-  default     = "https://charts.bitnami.com/bitnami"
-  description = "Helm repository"
-}
-
-variable "helm_create_namespace" {
-  type        = bool
-  default     = true
-  description = "Whether to create k8s namespace with name defined by `namespace`"
-}
-
-variable "namespace" {
-  type        = string
-  default     = "kube-system"
-  description = "The K8s namespace in which the external-dns will be installed"
-}
-
-variable "rbac_create" {
-  type        = bool
-  default     = true
-  description = "Whether to create and use RBAC resources"
-}
-
-variable "service_account_create" {
-  type        = bool
-  default     = true
-  description = "Whether to create Service Account"
-}
-
-variable "service_account_name" {
-  type        = string
-  default     = "external-dns"
-  description = "The k8s external-dns service account name"
-}
-
-variable "irsa_role_create" {
-  type        = bool
-  default     = true
-  description = "Whether to create IRSA role and annotate service account"
-}
-
-variable "irsa_role_name_prefix" {
-  type        = string
-  default     = "external-dns-irsa"
-  description = "The IRSA role name prefix for external DNS"
-}
-
-variable "irsa_assume_role_enabled" {
-  type        = bool
-  default     = false
-  description = "Whether IRSA is allowed to assume role defined by assume_role_arn. Useful for hosted zones in another AWS account."
+  description = "Set to false to prevent the module from creating any resources."
 }
 
 variable "irsa_assume_role_arn" {
```

Comment:
This variable was replaced by `irsa_assume_role_arns` (list). We should use the new one instead.